airframe-sql/src/main/scala/wvlet/airframe/sql/analyzer/TypeResolver.scala

```diff
@@ -441,9 +441,9 @@ object TypeResolver extends LogSupport {
 
     val results = expr match {
       case i: Identifier =>
-        lookup(i.value).map(toResolvedAttribute(i.value, _)).distinct
+        lookup(i.value).map(toResolvedAttribute(i.value, _))
```

Comment:
All tests passed but I'm still wondering if this change has any downsides. 🤔

Reply:
I think these distinct methods were introduced when we had no MultiSourceColumn.